config/environments/test.rb

```diff
@@ -70,4 +70,8 @@
 
   # Annotate rendered view with file names.
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  # We use example.com in our tests, so some redirects may be flagged as unsafe.
+  # In other environments, we should not expect this, so just override the setting here.
+  config.action_controller.raise_on_open_redirects = false
```

Comment:
I'm not sure I _love_ this solution, but I checked and changing this would mean either allowing open redirects for certain things in the code regardless of environment (yuck) or changing a lot of tests. For now, I think this is good enough

Reply:
I may have a better solution: https://github.com/Bilka2/otwarchive/commit/ecaa03b5e5788f024b67ee4b7b56795098a79f28

Coding credit goes to ticking instant, it's based on this branch: https://github.com/tickinginstant/otwarchive/tree/noodling-rails-7